geo_features.js

```diff
@@ -142,6 +143,32 @@ function featureStyle(feature) {
     strokeWeight = 1.5;
     strokeDash   = '1';
     fillColor    = '#031056';
+  } else if(entity1name == 'Roman Empire') {
```

Comment:
ooh, now that I'm seeing these here, I'm thinking the corresponding `ohmec_data_<>.js` file should be the one to define these. that way they're in the author's control, not a random disassociated file. Will file an issue but leave for now.